core/vm/actor/builtin/v0/miner/policy.hpp

```diff
@@ -19,7 +19,7 @@ namespace fc::vm::actor::builtin::v0::miner {
   using primitives::TokenAmount;
   using primitives::sector::RegisteredProof;
 
-  constexpr size_t kEpochDurationSeconds{25};
+  constexpr size_t kEpochDurationSeconds{30};
```

Comment:
kEpochDurationSeconds  is already declared in "const.hpp".

Reply:
Please, take a look at "core/clock/chain_epoch_clock.hpp:27" at `kEpochDuration{15}`. Is it correct value?

Reply:
Miner's `kEpochDurationSeconds` is constexpr, but "const.hpp" could be mutable (using config), so i'll leave it for now